src/RequestHandler/outofprocess/forwardinghandler.cpp

```diff
@@ -523,6 +525,23 @@ REQUEST_NOTIFICATION_STATUS
 
     default:
         DBG_ASSERT(m_RequestStatus == FORWARDER_DONE);
+        if (m_hRequest == NULL && m_pWebSocket == NULL)
```

Comment:
m_fHttpHandleInClose is set before closing wihttp handle (i.e., winhttp handle closing callback has not been triggered yet). We need do this NULL check to make sure the handle closing callback was called and handle was set to null so that there will be no more callback from WinHttp.